ocfweb/docs/templates/docs/partials/host.html

```diff
@@ -24,6 +24,24 @@
             {% endif %}
         </p>
     </div>
+    <div class="col-sm-12 ssh-keys">
+        {% if host.has_ssh_rsa_key %}
+          {% if host.ssh_rsa_key == None %}
+              <p>This machine does not have an SSH RSA Key</p>
+          {% else %}
+              <a href="#sshrsa{{host.hostname}}">Show SSH RSA Key: </a><p class="expand" id="sshrsa{{host.hostname}}">{{host.ssh_rsa_key}}</p>
+          {% endif %}
+        {% endif %}
+    </div>
+    <div class="col-sm-12 ssh-keys">
+            {% if host.has_ssh_dsa_key %}
+            {% if host.ssh_dsa_key == None %}
+                <p>This machine does not have an SSH DSA Key</p>
```

Comment:
imo, If it doesn't have a key of one type, it should just show nothing.

Reply:
interesting, easy fix, but why do you think so?

Reply:
from #437 jvperrin mentioned that we don't want the `sshfp` facts since we don't use them